frontend/src/components/ExperimentCollection/ExperimentCollectionDashboard/ExperimentCollectionDashboard.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { API_ROOT } from "../../../config";
+import ExperimentCollection from "@/interfaces/ExperimentCollection.interface";
+
+
+interface ExperimentCollectionDashboardProps {
+    experimentCollection: ExperimentCollection;
+}
+
+export const ExperimentCollectionDashboard: React.FC<ExperimentCollectionDashboardProps> = ({ experimentCollection }) => {
+
+    const dashboard = experimentCollection?.dashboard || [];
+
+    // TODO: get next experiment and about link from experimentCollection
+    const nextExperiment = experimentCollection.next_experiment; // TODO: get next_experiment from experimentCollection
+    const aboutContent = experimentCollection.about_content;
+
+    return (
+        <>
+            <div className="hero">
+                <div className="intro">
+                    <p>{experimentCollection?.description}</p>
+                    <nav className="actions">
+                        {nextExperiment && <a className="btn btn-lg btn-primary" href={"/" + nextExperiment.slug}>Volgende experiment</a>}
+                        {aboutContent && <Link className="btn btn-lg btn-outline-primary" to={`/collection/${experimentCollection.slug}/about`}>Over ons</Link>}
+                    </nav>
+                </div>
+                <div className="results">
```

Comment:
What's this empty div doing?

Reply:
You mean this is a placeholder for the score? In that case, we probably want it to be called "score" rather than "results". But then it can be left for later.